tests/python/gaia-ui-tests/gaiatest/tests/functional/marketplace/test_marketplace_packaged_app.py

```diff
@@ -34,7 +34,7 @@ def test_search_and_install_app(self):
         confirm_install = ConfirmInstall(self.marionette)
         confirm_install.tap_confirm()
 
-        self.assertEqual(self.apps.displayed_app.name, 'Marketplace')
+        self.assertEqual(self.apps.displayed_app.manifest_url, 'https://marketplace.firefox.com/packaged.webapp')
```

Comment:
You could use marketplace.manifest_url here or Marketplace(self.marionette).maniste_url (first one is preferred if possible).